content/e20co/drone-marathon/index.md

```diff
@@ -78,7 +78,19 @@ Will result in:
 
 # Secrets
 
-Not Implemented.
+The Marathon plugin supports reading credentials from the Drone secret store. This is highly recommended instead of storing credentials in the pipeline configuration in plain text.
+
+```diff
+pipeline:
+  marathon:
+    image: e20co/drone-marathon:0.5
+    server: http://marathon.mesos:8080
+    values:
+-     DATABASE_PASSWORD: SomeSecretPassword
++     DATABASE_PASSWORD: ${DATABASE_PASSWORD}
```

Comment:
Ok.  This might work then, assuming that a user does the following:

* `drone secret add --image=e20co/drone-marathon <repo> MY_SECRET mySecretValue`
* inside the `.drone.yml` they reference `${MY_SECRET}` which resolves to `mySecretValue` (because it's been injected into the environment)
* The module continues, now that the secret has been populated.

Or am I misunderstanding how this actually works?